### PR TITLE
Customizing Django Admin

### LINKED ISSUE #1082: Customizing Django Admin
### Brief Summary: 
The Django framework comes with its own admin templates.With some know-how its possible to customize it into suiting your project even more.

### Key Takeaways: 
1. How to access the Django admin page.
2. Identifying some of the features that can be  customized.
3. Customizing the Django admin to suit their project.

### References: 
N/A

### PR DESCRIPTION
**Pull request by Sajeyks**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you placed your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

fixes #1082

### CODE REVIEW

**Comment by geoffrey45** [comment]
@Sajeyks Please complete your PR info. ie. The title and checkboxes. Also, don't forget to close the previous pull request.

**Comment by Sajeyks** [comment]
This fixes #1082

File: articles/customizing-django-admin/index.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+###  2. Changing the Django administration header text
+To change the admin site header-text,login page, and the HTML title tag to our bookstore's instead, add the following code in `urls.py`.
+```python
+#...
+admin.site.site_header  =  "Custom bookstore admin"  
+admin.site.site_title  =  "Custom bookstore admin site"
+admin.site.index_title  =  "Custom Bookstore Admin"
+```
+*site_header* changes the *Django administration* text which appears on the login page. The *site_title* appears on the admin page after we have logged in. The *index_title* changes our site's title that appears on the top of the page.
+
```
Comment (geoffrey45) [inline articles/customizing-django-admin/index.md:167]: The `site_title` is does not appear on the admin page as you have indicated above. Please see [this](https://docs.djangoproject.com/en/dev/ref/contrib/admin/#adminsite-attributes) document and fix your explanations. Also, write the 'site_title', etc as inline code blocks. ie. \`site_title`

**Review by geoffrey45 — CHANGES_REQUESTED** [review]
@Sajeyks I'm almost done with the tech review. Just some minor issues. Please fix them and ping me when you are done

**Pull request closed without merging**